doc/release-process.md

```diff
@@ -203,7 +203,20 @@ Codesigner only: Sign the macOS binary:
     tar xf bitcoin-osx-unsigned.tar.gz
     ./detached-sig-create.sh -s "Key ID"
     Enter the keychain password and authorize the signature
-    Move signature-osx.tar.gz back to the gitian host
+
+Now a manual deterministic disk image (dmg) creation is required (gbuilt with `gitian-osx-signer.yml` while having the signatures.tar.gz file in the inputs)
+
+notarize the disk image:
+
+    xcrun altool --notarize-app --primary-bundle-id "org.bitcoinfoundation.Bitcoin-Qt" -u "<code-signer-apple-developer-account-username>" -p "<password>" --file bitcoin-${VERSION}-osx.dmg
+
+The notarization takes a few minutes. Check the status:
+
+    xcrun altool --notarization-info <RequestUUID-from-notarize-app-step> -u "<code-signer-apple-developer-account-username>" -p "<password>"
+
+Staple the notarization ticket onto the application
+
+    xcrun stapler staple dist/Bitcoin-Qt.app
```

Comment:
Can `gitian-osx-signer.yml` do this, so we have gitian signatures on the final DMG?

Reply:
I don't think so since `stapler` is an macOS developer tool. Unsure if this can be made portable.

But, we have gitian signatures on the final dmg. Everything remains deterministic. It's just the code signature and the notarization ticket that needs to be made manual and centralised.
Though the notarization ticket is bound to the hash of the dmg (which is built deterministic).

I think t is pointless for the macOS code signing key holder to notarize something else because the .dmg hash would not match and the notarization check would fail.

Reply:
So we're NOT planning to upload the stapled DMG anywhere?

Reply:
We do.
The staple command above does create the `CodeResources` file (that actually contains the notarization ticket). The macOS code signer commits that file together with the code signatures. See this example: https://github.com/jonasschnelli/bitcoin-detached-sigs/tree/0.19/osx/dist/Bitcoin-Qt.app/Contents.

The `detached-sig-apply.sh` takes also that CodeResource file and places it in the final deterministic created app-folder that end up in the final dmg.

Reply:
It's a bit confusing that there are two `CodeResources` files, that might be worth documenting. There's on in `Bitcoin-Qt.app/Contents/` that you copy below, that's new. And then there's one in `_CodeSignature`, which is bigger.